tools/standard-test/setup_limits.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# This is necessary for servers that try to run OpenSearch with docker
+# The hosts have to have these settings ready
+# Or the OpenSearch Process will not run
+sudo sysctl -w vm.max_map_count=262144
+ulimit -n 65535
+# Required for PA and similar processes that require shared memory to work
+sudo chmod -R 777 /dev/shm
+
+# As for docker the host must have these as well in /etc/sysconfig/docker and restart service
+# OPTIONS="--default-ulimit nofile=65535:65535"
```

Comment:
Not required? Can we remove

Reply:
This is the requirements for LINUX host, I have already move these comments to the top of the file.